src/keboola_mcp_server/tools/jobs.py

```diff
@@ -171,71 +175,149 @@ class ListJobsOutput(BaseModel):
+
+    CONTEXT:
+    Jobs in Keboola are execution records of components (extractors, transformations, writers, flows).
+    Each job represents a single run with its status, timing, configuration, and results.
+
+    TWO MODES OF OPERATION (controlled by job_ids parameter):
+
+    MODE 1: GET DETAILS FOR SPECIFIC JOBS (job_ids is non-empty)
+    - Provide one or more job IDs: job_ids=["12345", "67890"]
+    - Returns: FULL details for each job including status, config_data, results, timing, and metadata
+    - Ignores: All filtering/sorting parameters (status, component_id, config_id, limit, offset, sort_by, sort_order)
+    - Use when: You know specific job IDs and need complete information about them
+
+    MODE 2: LIST/SEARCH JOBS (job_ids is empty)
+    - Leave job_ids empty: job_ids=[]
+    - Returns: SUMMARY list of jobs (id, status, component_id, config_id, timing only - no config_data or results)
+    - Supports: Filtering by status/component_id/config_id, pagination with limit/offset, sorting
+    - Use when: You need to find jobs, see recent executions, or monitor job history
+
+    DECISION GUIDE:
+    - Start with MODE 2 (list) to find jobs → then use MODE 1 (details) if you need full information
+    - If you already know job IDs → use MODE 1 directly
+    - For monitoring/browsing → use MODE 2 with filters
+    - For troubleshooting specific jobs → use MODE 1 with known job IDs
+
+    COMMON WORKFLOWS:
+    1. Find failed jobs: job_ids=[], status="error" → identify problematic job IDs → get details with MODE 1
+    2. Check recent runs: job_ids=[], component_id="...", limit=10 → see latest executions
+    3. Monitor specific job: job_ids=["123"] → poll for status and results
+    4. Troubleshoot config: job_ids=[], config_id="...", status="error" → find which runs failed
 
     EXAMPLES:
-    - If status = "error", only jobs with status "error" will be listed.
-    - If status = None, then all jobs with arbitrary status will be listed.
-    - If component_id = "123" and config_id = "456", then the jobs for the component with id "123" and configuration
-      with id "456" will be listed.
-    - If limit = 100 and offset = 0, the first 100 jobs will be listed.
-    - If limit = 100 and offset = 100, the second 100 jobs will be listed.
-    - If sort_by = "endTime" and sort_order = "asc", the jobs will be sorted by the end time in ascending order.
-    """
-    _status = [status] if status else None
 
+    MODE 1 - Get full details:
+    - job_ids=["12345"] → detailed info for job 12345
+    - job_ids=["12345", "67890"] → detailed info for multiple jobs
+
+    MODE 2 - List/search jobs:
+    - job_ids=[] → list latest 100 jobs (default)
+    - job_ids=[], status="error" → list only failed jobs
+    - job_ids=[], status="processing" → list currently running jobs
+    - job_ids=[], component_id="keboola.ex-aws-s3" → list jobs for S3 extractor
+    - job_ids=[], config_id="12345" → list jobs for specific configuration
```

Comment:
The same here, shouldn't this rather be:

```suggestion
    - job_ids=[], component_id="keboola.ex-aws-s3", config_id="12345" → list jobs for specific S3 extractor's configuration
```